jest.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   roots: [
     'src'
   ],
+  maxWorkers: 1,
```

Comment:
Multiple workers -> concurrent test cases running, meaning if a user was added to the DB in a post test case, it can mess up user test cases :( I don't think this is the best testing practices but idk what the alternative is